CHANGES.rst

```diff
@@ -1,3 +1,589 @@
+
+- Cloning a cosmology with changed parameter(s) now appends "(modified)" to the
+  new instance's name, unless a name is explicitly passed to ``clone``. [#11536]
+
+- Allow ``m_nu`` to be input as any quantity-like or array-like -- Quantity,
+  array, float, str, etc. Input is passed to the Quantity constructor and
+  converted to eV, still with the prior mass-energy equivalence enabled. [#11640]
+
+astropy.io.fits
+^^^^^^^^^^^^^^^
+
+- For conversion between FITS tables and astropy ``Table``, the standard mask
+  values of ``NaN`` for float and null string for string are now properly
+  recognized, leading to a ``MaskedColumn`` with appropriately set mask
+  instead of a ``Column`` with those values exposed. Conversely, when writing
+  an astropy ``Table`` to a FITS tables, masked values are now consistently
+  converted to the standard FITS mask values of ``NaN`` for float and null
+  string for string (i.e., not just for tables with ``masked=True``, which no
+  longer is guaranteed to signal the presence of ``MaskedColumn``). [#11222]
+
+astropy.io.votable
+^^^^^^^^^^^^^^^^^^
+
+- The use of ``version='1.0'`` is now fully deprecated in constructing a ``astropy.io.votable.tree.VOTableFile``. [#11659]
+
+astropy.modeling
+^^^^^^^^^^^^^^^^
+
+- Removed deprecated ``astropy.modeling.blackbody`` module. [#10972]
+
+astropy.table
+^^^^^^^^^^^^^
+
+- Added ``Column.value`` as an alias for the existing ``Column.data`` attribute.
+  This makes accessing a column's underlying data array consistent with the
+  ``.value`` attribute available for ``Time`` and ``Quantity`` objects. [#10962]
+
+- In reading from a FITS tables, the standard mask values of ``NaN`` for float
+  and null string for string are properly recognized, leading to a
+  ``MaskedColumn`` with appropriately set mask. [#11222]
+
+- Changed the implementation of the ``table.index.Index`` class so instantiating
+  from this class now returns an ``Index`` object as expected instead of a
+  ``SlicedIndex`` object. [#11333]
+
+astropy.units
+^^^^^^^^^^^^^
+
+- The ``physical_type`` attribute of units now returns an instance of
+  ``astropy.units.physical.PhysicalType`` instead of a string.  Because
+  ``PhysicalType`` instances can be compared to strings, no code changes
+  should be necessary when making comparisons.  The string representations
+  of different physical types will differ from previous releases. [#11204]
+
+- Calling ``Unit()`` with no argument now returns a dimensionless unit,
+  as was documented but not implemented. [#11295]
+
+astropy.utils
+^^^^^^^^^^^^^
+
+- Removed deprecated ``utils.misc.InheritDocstrings`` and ``utils.timer``. [#10281]
+
+- Removed usage of deprecated ``ipython`` stream in ``utils.console``. [#10942]
+
+astropy.wcs
+^^^^^^^^^^^
+
+- Deprecate ``accuracy`` argument in ``all_world2pix`` which was mistakenly
+  *documented*, in the case ``accuracy`` was ever used. [#11055]
+
+Bug Fixes
```

Comment:
Used to have two empty lines between the 3 main sections

Reply:
Thanks, will fix

Reply:
Actually, I see where to do this in the template, so I'll just do that at the same time.